posts/posts-router.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const db = require('../data/db');
+const port = 4001;
+const router = express.Router();
+
+router.post("/", (req, res) => {
+    const postInfo = req.body;
+  
+    if (!postInfo.title || !postInfo.contents) {
+      res.status(400).json({
+          success: false,
+          errorMessage: "Please provide name and bio for the user."
+        });
+      }
+    db
+      .insert(postInfo)
+      .then(u => {
```

Comment:
```suggestion
      .then(post => {
```
Pleasse be more semantic with your variable names. It is not immediately clear what `u` means to other developers.